pkg/harvester/config/harvester-map.js

```diff
@@ -41,7 +41,9 @@ export const ACCESS_CREDENTIALS = {
   INJECT_SSH: 'sshPublicKey'
 };
 
-export const RunStrategys = ['Always', 'RerunOnFailure', 'Manual', 'Halted'];
+export const runStrategies = ['Always', 'RerunOnFailure', 'Manual', 'Halted'];
+
+export const maintenanceStrategies = ['Migrate', 'RestartOnEnable', 'RestartOnDisable', 'StayOff'];
```

Comment:
Please check the [HEP](https://github.com/harvester/harvester/pull/5984) for more information about how the strategies are named.